hicommonwealth/commonwealth
# Update Community Id (Url)

## Link to Issue
Closes: #6849

## Description of Changes
- Sets up a new `/communities/update_id` route + handler + function
- The new function finds all references of the original community id and updates them to the new community id

## Test Plan
- Update the name of various popular communities from the admin-panel e.g. dYdX, 1inch, etc
- Ensure the following works after changing the id:
  - thread/comment/reaction creation
  - community joining
  - community buy/sell stakes
  - promote/demote users
  - view analytics for community

## Deployment Plan

1. 

## Other Considerations

- Redirects separated into another ticket: https://github.com/hicommonwealth/commonwealth/issues/6863
